src/Kunstmaan/FixturesBundle/Builder/PageBuilder.php

```diff
@@ -220,7 +220,7 @@ private function ensureUniqueUrl(NodeTranslation $translation, HasNodeInterface
         $translationWithSameUrl = $this->nodeTranslationRepo->getNodeTranslationForUrl($translation->getUrl(), $translation->getLang(), false, $translation);
 
         if ($translationWithSameUrl instanceof NodeTranslation) {
-            $translation->setSlug($this->slugifier->slugify($this->incrementString($translation->getSlug())));
+            $translation->setSlug($this->slugifier->slugify(static::incrementString($translation->getSlug())));
```

Comment:
This method has no reason to be static in the first place. I would suggest changing the method definition instead of this line. It is private anyway so you always need an instantiated object of this class so the static adjective is useless.